apps/fyler/src/fyler_server.erl

```diff
@@ -25,7 +25,7 @@
 %% API
 -export([start_link/0]).
 
--export([run_task/3, clear_stats/0, pools/0, current_tasks/0, send_response/3, authorize/2, is_authorized/1, tasks_stats/0, tasks_stats/1, save_task_stats/1, task_status/1, cancel_task/1]).
+-export([run_task/3, clear_stats/0, pools/0, current_tasks/0, send_response/2, authorize/2, is_authorized/1, tasks_stats/0, tasks_stats/1, save_task_stats/1, task_status/1, cancel_task/1]).
```

Comment:
Line 28 is too long: -export([run_task/3, clear_stats/0, pools/0, current_tasks/0, send_response/2, authorize/2, is_authorized/1, tasks_stats/0, tasks_stats/1, save_task_stats/1, task_status/1, cancel_task/1])..